front/app/modules/commercial/content_builder/admin/components/ContentBuilderFrame/index.tsx

```diff
@@ -35,6 +35,8 @@ const ContentBuilderFrame = ({ projectId }: { projectId: string }) => {
         canvas
         style={{
           padding: '4px',
+          width: '100%',
+          maxWidth: '1166px',
```

Comment:
This is such a magic number - can we make it 1050px as before (given that it will only affect the actual content builder page)?

Reply:
Ok - '1166px' came from Ben. Should I double check with him first if there is a reason for the specific value?

Reply:
The value is coming from `containers/ProjectsShowPage/styles.ts` so it will be respected in the project page. No need to specify it in the content builder, as long as the width of the editor description is `100%`.

Reply:
I ended up needing to make it 1000px as before (but now so it only affects the builder page) so that there is a small gap between the content and the Settings panel:
![SettingsGap](https://user-images.githubusercontent.com/33987955/167779976-0be72956-0c59-4ab5-bb5f-d19855e61a2c.png)

Reply:
Whether this gap is there or not will depend on your screen size :) But it's ok! Shall we keep this width and the max width for the root node consistent, though - make them both 1000 or 1050?